### ISSUE TITLE
Improvements for project quality overview

### ISSUE THREAD

**Issue by ewan-escience** [research-software-directory/RSD-as-a-service#804]
In this issue we can collect feedback on the project quality overview (#764), implemented in #790. Some requests we have already gotten: 

- For any of the numeric scores (1,2,3, ..), it is hard to say if it's good or bad. however, a '0' is always bad. could a zero also be a red 'X' to make it stand out more? (Jisk)
- Can the columns be made sortable? I have more than 50 projects, to have newest one on top (or ones with specific isssues) would be useful. (Jisk) @jiskattema do you mean to sort the rows? That is already possible, but not (yet) on creation date of the project.
- Add a column with the Grant ID strings (Rena)
- Add the option to reorder the columns (Rena)

Would you also want a similar overview for software pages?

**Comment by dmijatovic** [comment]
@jiskattema en @ewan-escience Concerning numeric scores. Would it be sufficient to have only cross/checkmark sign for these scores too?

**Comment by jiskattema** [comment]
> @jiskattema do you mean to sort the rows? That is already possible, but not (yet) on creation date of the project.

I find it always confusing how to explain rows vs. columns. Lets say i want to sort by 'this project has a funder' to fix all the projects where the funder has not been set yet. Does that make it clearer? (I also had a talk with Jason who said that is already possible, i was only looking at the screenshot and didnt see the typical arrow).

**Comment by ewan-escience** [comment]
> I find it always confusing how to explain rows vs. columns. Lets say i want to sort by 'this project has a funder' to fix all the projects where the funder has not been set yet. Does that make it clearer? (I also had a talk with Jason who said that is already possible, i was only looking at the screenshot and didnt see the typical arrow).

Yes, that will be possible when we release this (and that's what I mean with sorting by row 😀). Rena, on the other hand, wants to drag and drop the columns, so that the columns she wants to see are close together.

**Comment by bakhshir** [comment]
I told you, @ewan-escience, we need Excel functionality! ;-)

**Comment by jiskattema** [comment]
> @jiskattema en @ewan-escience Concerning numeric scores. Would it be sufficient to have only cross/checkmark sign for these scores too?

I'm undecided there. For some projects we clearly have 2 funders, and having a '2' instead of a green checkmark makes it easier. Similarly, when quickly searching a project with 'much output' would be possible by sorting on # output in decreasing order. But this could be too niche..

**Comment by jiskattema** [comment]
Additional request:
 
Make the rows actual hrefs to the edit-project page. That is, in the table clicking on the project title will take you to the RSD page to edit the project. Alternatively, this could be via an extra column containing a clicky-icon

**Comment by jiskattema** [comment]
> Would you also want a similar overview for software pages?

I think yes, although i assume the TLs like @jmaassen would be using that more than me.
It keeps the software and projects more similar, so less surprises (good!) when using the RSD.

**Comment by jmaassen** [comment]
Yup, for the TLs we would want something similar for software. It may be interesting to do a quick prototype and the get some feedback.

**Comment by ewan-escience** [comment]
@jiskattema @bakhshir @nielsdrost @lopeztarifa We just made a release which includes the quality overview page. Login and go to  https://research-software-directory.org/user/project-quality to try it out.

**Comment by jiskattema** [comment]
Looks good!
first impressions feedback:
* 'Project quality overview' better called something like 'Project metadata overview'. It's not about the actual project content
* Column 'Complete' => 'Metadata score' ? complete suggests timewise completion, ie. weeks until end date.
* a column for count of  'related tools', and 'related projects' ?

**Comment by bakhshir** [comment]
I love it! I immediately could add all contact points to the projects page that had none. Also sorting of columns is really useful.

I still would like to have textual field for grantID, and start/end date. We had very creative previous RSD team, who assigned 1900 to the start date of some projects. I hope i combed it all out, but not entirely sure.

**Comment by jiskattema** [comment]
> I love it! I immediately could add all contact points to the projects pages that had none. Also sorting of columns is really useful.
> 
> I still would like to have textual field for grantID, and start/end date. We had very creative previous RSD team, who assigned 1900 to the start date of some projects. I hope i combed it all out, but not entirely sure.

maybe similar to my comment above wrt. numbers and red crosses?
If set, show the actual value, if not set, or none/null/... show red cross?
